setup.py

```diff
@@ -49,11 +49,10 @@
     python_requires=">=3.6",
     package_data={"myst_nb": ["_static/*"]},
     install_requires=[
-        "myst-parser~=0.13.3",
+        "myst-parser>=0.13.5",
         "docutils>=0.15",
         "sphinx>=2,<4",
-        # TODO 0.3.2 requires some changes to the pytests
-        "jupyter_sphinx==0.3.1",
+        "jupyter_sphinx>=0.3.2",
```

Comment:
```suggestion
        "jupyter_sphinx~=0.3.2",
```

we should still pin to <0.4

Reply:
Sounds good - I thought this was required, but it looks like the issue was elsewhere.